tests/assets/ansible/roles/caasp/tasks/main.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Add repos
+  include_tasks: '{{ item }}'
+  vars:
+    params:
+      files:
+        - "{{ ansible_os_family }}.yaml"
+        - default.yaml
```

Comment:
Where is this `default.yaml` file? I don't see it

Reply:
Initially this was supposed to be generic but now we have a caasp role this does't make sense. I removed the reference to it (thoug this would not fail ansible in the given code)